tests/utils/test_cache_utils.py

```diff
@@ -174,61 +174,54 @@ def test_static_cache_exportability(self):
         """
         Tests that static cache works with `torch.export()`
         """
-        import torch
-
-        if version.parse(torch.__version__) < version.parse("2.3"):
+        if not is_torch_greater_or_equal_than_2_3:
             self.skipTest(reason="This test requires torch >= 2.3 to run.")
 
+        set_seed(0)
         device = "cpu"
         dtype = torch.float32
```

Comment:
Add copyright here in this file? 

@ArthurZucker , @amyeroberts 

What's your general guideline of having a file with mixed credits?

Reply:
Found one with mixed credits (https://github.com/huggingface/transformers/blob/d7b04ea14ddfef04fc110f360e8ea81cfcd1c86a/src/transformers/integrations/ggml.py#L2). Not sure if it's the standard way